src/sentry/api/bases/external_actor.py

```diff
@@ -23,27 +23,35 @@ class ExternalActorSerializerBase(CamelSnakeModelSerializer):  # type: ignore
     external_id = serializers.CharField()
     external_name = serializers.CharField(required=True)
     provider = serializers.ChoiceField(choices=get_provider_choices(AVAILABLE_PROVIDERS))
+    integration_id = serializers.IntegerField(required=False, allow_null=True)
 
     @property
     def organization(self) -> Organization:
         return self.context["organization"]
 
-    def get_actor_id(self, validated_data: MutableMapping[str, Any]) -> int:
-        return int(validated_data.pop(self._actor_key).actor_id)
+    def validate_integration_id(self, integration_id: str) -> Optional[str]:
+        if not integration_id:
+            return None
```

Comment:
I think we could just have `return` here

Reply:
@MeredithAnya hmm is it not better to be explicit with None?